uaclient/util.py

```diff
@@ -358,6 +358,25 @@ def get_platform_info() -> 'Dict[str, str]':
     return platform_info
 
 
+def apply_series_overrides(orig_access: 'Dict[str, Any]') -> None:
+    """Apply series-specific overrides to an entitlement dict.
```

Comment:
I'd like the docstring to be explicit about mutation (and probably call out the removal of the `series` key too).